ios/Transitioning/REATransitionAnimation.m

```diff
@@ -42,8 +42,10 @@ + (REATransitionAnimation *)transitionWithAnimation:(CAAnimation *)animation
 
 - (void)play
 {
+  if (_delay > 0){
```

Comment:
I'm fine with merging this change in. However, I'd like to have a comment here explaining why this check has been introduced (basically what you explain in the PR description).

I also wonder what is the delay in case we use CACurrentMediaTime(). Is this like 1 frame delay (16ms) or more than that? Also if you set delay to be 60 ms is it going to start after 60ms or after 60ms + 1 frame